apps/mobile-alphabet/src/app/Components/Menu-detail.presenter.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Text, View } from 'react-native';
+import config from './Config.json';
+import { AlphabetData } from './Menu';
+
+export function MenuDetialScreen() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    const [selectedLetterSequenceNumber, setSelectedLetterSequenceNumber] = useState<number>(1);
+
+    useEffect(() => {
+        //TODO cache this after fetching it once
+        const fetchData = async () => {
+            try {
+                //TODO use context api
+                const response = await fetch(config.apiUrl, {
+                    mode: 'cors',
+                });
+                setAlphabetData(await response.json());
+            } catch (error) {
+                console.error('Error fetching data:', error);
+            }
+        };
+
+        fetchData();
+    }, []);
+
+    if (!alphabetData) {
+        return <Text>Loading...</Text>;
```

Comment:
I imagine that if there's a server error you just see this loader indefinitely right now.